.gitignore

```diff
@@ -37,6 +37,7 @@ doxygen/
 /keyboards/*/*/*/keymaps/Makefile
 /keyboards/*/*/*/*/keymaps/Makefile
 /keyboards/*/*/*/*/*/keymaps/Makefile
+secrets.h
```

Comment:
This should be in ` .git/info/exclude`, not here, as this file is global, and ` .git/info/exclude` is local only.

Reply:
Removed secrets.h - Macros have all been removed. I missed this line during the cleanup. Thank you!